services/drupal/web/themes/epa_theme/source/_patterns/04-layouts/header/_header.scss

```diff
@@ -2,8 +2,6 @@
 // Styles for Header Layout.
 
 .l-header {
-  position: relative;
-  z-index: 2;
```

Comment:
@jbwagner Why are these lines removed? We noticed a new bug during the demo today where the active local task menu item was shown on top of dropdown menus.